Разметка

aykuli opened the pull request.

aykuli committed:
Add sections of promo, features, blog post and subscribe blocks in index.html

cybersunt commented on index.html:
```diff
@@ -67,11 +67,58 @@ <h2 class="visually-hidden">3 лучших предложения</h2>
 						</li>
 					</ul>
 					<ul class="slider-toggles">
-						<li><button class="btn-toggle" type="button" name="slider-toggle"></button></li>
```
У тебя из кнопок выбрана, значит ей нужно добавиь свой спецкласс. И не забудь добавить для каждой кнопки свое название, это для JS пригодиться

cybersunt commented on index.html:
```diff
@@ -67,11 +67,58 @@ <h2 class="visually-hidden">3 лучших предложения</h2>
+				<section class="offers">
+							<h2 class="visually-hidden">Промо-акции</h2>
+							<div class="offers-item">
+								<h3>Малинка даром!</h3>
```
Это не заголовок, а обычный текст, который выделен больше и жирнее, для привлечения внимания к промо-блоку. И где спецклассы для каждого блока, по ккоторы ты будешь отделять один блок от другого?

cybersunt commented on index.html:
```diff
@@ -67,11 +67,58 @@ <h2 class="visually-hidden">3 лучших предложения</h2>
+							<div class="offers-item">
+								<h3>Малинка даром!</h3>
+								<p>При покупке 2 кг любого фруктового мороженого, добавим в ваш заказ банку малинового варенья бесплатно.</p>
+								<a href="" class="btn">Хочу варенье!</a>
```
заглушки для ссылок не теряй

aykuli replied:
вот я думала, а зачем они? что с ними, что без них ссылка ведет себя одинаково - ведет на начало страницы, отличие только в том, что в строке адреса браузера появляется решетка или нет. Ты можешь меня просветить?

cybersunt commented on index.html:
```diff
@@ -67,11 +67,58 @@ <h2 class="visually-hidden">3 лучших предложения</h2>
+							</ul>
+						</section>
+						<section class="blog-subscribe">
+							<article class="blog">
```
section без заголовка остался

cybersunt commented on index.html:
```diff
@@ -67,11 +67,58 @@ <h2 class="visually-hidden">3 лучших предложения</h2>
+								<a href="">10 способов сервировки фруктовых щербетов к столу</a>
+							</article>
+							<div class="subscribe">
+								<form class="subscribe-form" action="#" method="get">
```
раз взялась за форму делай ее по ТЗ. В теге форм нужно задать action="#" method="get" согласто ТЗ. И куда делся текст из формы?

cybersunt commented on index.html:
```diff
@@ -67,11 +67,58 @@ <h2 class="visually-hidden">3 лучших предложения</h2>
+								<form class="subscribe-form" action="#" method="get">
+									<label for="subscribe">Подпишитесь на нашу сладкую рассылку и будете всегда в курсе всего самого вкусного, что у нас происходит. Обещаем не спамить и не слать всякой ненужной ерунды. Честно =)
+									</label>
+									<input id="subscribe" type="email" placeholder="Электронная почта">
```
куда name у инпута потерялся? и почему id="subscribe" у поля email?

aykuli replied:
подписываются же на рассылку) не логично?)

cybersunt reviewed (commented):
Посмотри еще раз лекцию, фрагмент насчет заголовков, когда и где их стоит применять и для всех заглушек у картинок пропиши width и height. В ТЗ тоже смотри, когда верстаешь. У тебя некоторые моменты сейчас совсем не по ТЗ

aykuli committed:
amends in index.html in offers & blog-subscribe sections

1) added btn classes "toggle1", "toggle2", "toggle3" for toogles in slider,
2) changed h2 to p in offers-item block
3) added specigic classe for each block in offers-item
4) href="#"
5) added h2 for blog-cubscribe section
6) correct action="https://echo.htmlacademy.ru" method="post"
7) added name for input in subscribe section

cybersunt commented on catalog.html:
```diff
@@ -28,19 +28,19 @@
 					<li class="user-navigation-item">
 						<a class="btn-search" href="#">
-							<img src="http://placehold.it/" alt="">
+							<img src="http://placehold.it/" alt="" width="" height="">
```
а размер писать не нужно?

aykuli replied:
я ж пока не знаю, какие размеры. Как будут img, так меняю по факту. А заранее какие проставлять? Какие придумаю?

cybersunt replied:
открываешь макет, смотришь в нем какие размеры у картинке, которая будет на этом месте и вставляешь их

aykuli committed:
added img width height value and alt
